tests/sandbox/sandbox.sh

```diff
@@ -41,4 +41,5 @@ CURL_CLIENT=ON \
     --gov-script "${GOV_SCRIPT}" \
     --ledger-chunk-bytes 5MB \
     --label sandbox \
+    -n 127.0.0.1:8000 \
```

Comment:
Note that this prevents overriding the default since `-n` is additive.

Reply:
That's true, but unfortunately it's difficult to avoid if we want to have the default be 443 the rest of the time, which we do.

A possible solution would be to only add this line if $@ does not contain -n, and to omit it otherwise.

Reply:
If we change `-n` to be of format `-n foo bar` then it's easy to override.

Reply:
True, but it's a big change of style, we use this for all our options here and elsewhere (member info etc), I doubt the benefit offsets the user update costs.

Reply:
I think preventing overriding is a major issue. I have an alternative approach, will open a PR shortly.

Reply:
#1756